template.yaml

```diff
@@ -41,3 +41,17 @@ Resources:
     Properties:
       Name: Explode
       FunctionName: !GetAtt MacroFunction.Arn
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: "cfn-explode-macro"
+    Description: "macro that provides a template-wide Explode property for CloudFormation resources"
+    Author: "zaro0508"
+    SpdxLicenseId: "Apache-2.0"
+    # paths are relative to .aws-sam/build directory
+    LicenseUrl: "LICENSE"
+    ReadmeUrl: "README.md"
+    Labels: ["serverless", "template"]
```

Comment:
updates with labels specific to this lambda.  maybe add `serverless`, `macro`, and `cloudformation`